Rex.podspec

```diff
@@ -0,0 +1,136 @@
+#
+#  Be sure to run `pod spec lint Rex.podspec' to ensure this is a
+#  valid spec and to remove all comments including this before submitting the spec.
+#
+#  To learn more about Podspec attributes see http://docs.cocoapods.org/specification.html
+#  To see working Podspecs in the CocoaPods repo see https://github.com/CocoaPods/Specs/
+#
+
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  These will help people to find your library, and whilst it
+  #  can feel like a chore to fill in it's definitely to your advantage. The
+  #  summary should be tweet-length, and the description more in depth.
+  #
+
+  s.name         = "Rex"
+  s.module_name  = "Rex"
+  s.version      = "0.9.0-alpha.4"
+  s.summary      = "ReactiveCocoa Extensions"
+
+  s.description  = <<-DESC
+                   Extensions for ReactiveCocoa that may not fit in the core framework.
+                   DESC
+
+  s.homepage     = "https://github.com/neilpa/Rex"
+  # s.screenshots  = "www.example.com/screenshots_1.gif", "www.example.com/screenshots_2.gif"
+
+
+  # ―――  Spec License  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Licensing your code is important. See http://choosealicense.com for more info.
+  #  CocoaPods will detect a license file if there is a named LICENSE*
+  #  Popular ones are 'MIT', 'BSD' and 'Apache License, Version 2.0'.
+  #
+
+  s.license      = "MIT"
+  # s.license      = { :type => "MIT", :file => "FILE_LICENSE" }
+
+
+  # ――― Author Metadata  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Specify the authors of the library, with email addresses. Email addresses
+  #  of the authors are extracted from the SCM log. E.g. $ git log. CocoaPods also
+  #  accepts just a name if you'd rather not provide an email address.
+  #
+  #  Specify a social_media_url where others can refer to, for example a twitter
+  #  profile URL.
+  #
+
+  s.author             = { "Neil Pankey" => "npankey@gmail.com" }
+  # Or just: s.author    = "Bjarke Hesthaven Søndergaard"
+  # s.authors            = { "Bjarke Hesthaven Søndergaard" => "bhs@trifork.com" }
+  # s.social_media_url   = "http://twitter.com/Bjarke Hesthaven Søndergaard"
+
+  # ――― Platform Specifics ――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  If this Pod runs only on iOS or OS X, then specify the platform and
+  #  the deployment target. You can optionally include the target after the platform.
+  #
+
+  # s.platform     = :ios
+  # s.platform     = :ios, "8.0"
+
+  #  When using multiple platforms
+  s.ios.deployment_target = "8.0"
+  s.osx.deployment_target = "10.10"
+  # s.watchos.deployment_target = "2.0"
```

Comment:
Are there any valid deployment targets for Rex? It'd be cool to include those, too 🎉